man/man1/abbr.1

```diff
@@ -218,7 +218,7 @@ Behave as if `--quiet` was passed? (0 or 1, default 0)
 
 .IP \(bu
 \fIABBR_USER_ABBREVIATIONS_FILE\fR
-File abbreviations are stored in (default ${HOME}/.config/zsh/abbreviations)
+File abbreviations are stored in (default ${HOME}/.config/zsh-abbr/user-abbreviations)
```

Comment:
How about copying over from https://zsh-abbr.olets.dev/configuration-variables.html


```suggestion
Path to the file user abbreviation are stored in (default ${XDG_CONFIG_HOME:-$HOME/.config}/zsh-abbr/user-abbreviations, with legacy support for using ${XDG_CONFIG_HOME:-$HOME/.config}/zsh/abbreviations instead if a file exists at that path)
```

Reply:
Looks good! By the way, isn't it abbreviation**s**, not abbreviation?

Reply:
Good catch!

"the file user" parses weird too. How about

```diff
- Path to the file the user abbreviation
+ Path to the file the user abbreviations
```

If you want, you're welcome to make that correction in https://github.com/olets/zsh-abbr-v6-docs also!

Reply:
Sure!